reference-client/pom.xml

```diff
@@ -26,7 +26,7 @@
     <dependency>
       <groupId>com.transformuk.hee</groupId>
       <artifactId>reference-api</artifactId>
-      <version>2.10.1</version>
+      <version>2.10.2</version>
```

Comment:
In my opinion, we should bump version of reference-client as well, so that TCS/Bulk upload services can reference this new version.